CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -27,7 +29,7 @@ public IsAccountActiveMiddleware(RequestDelegate next)
         /// <summary>
         /// 
         /// </summary>
```

Comment:
If you're not planning to add any summary here - please, remove these lines. Don't just leave empty summary block like this.

Reply:
Solved in https://github.com/ita-social-projects/WhatBackend/pull/372/commits/a464d5562ac8934f2672925f553ab20e2a315825